Bugfix/modal&up button

marinakravchuk opened the pull request:
Modal window was updated by next changes:  
1. Background collor 
2. Videos were deleted and chaged to text 
3.  "Buy now " button  has collor  changes. svg and animation were added
 4. Scroll bar was customized

marinakravchuk committed:
small fix

marinakravchuk committed:
second update

marinakravchuk committed:
Merge branch 'main' into bugfix/modal&up_button

marinakravchuk committed:
update animation

Andrew-aka-Lemon reviewed (commented):
виглядає добре
додай тільки cursor pointer при hover на кнопку buy now
і додай transition для тіні карток, щоб плавненько вони з'являлись

marinakravchuk committed:
cursor and  transition

додай тільки cursor pointer при hover на кнопку buy now

1:09
і додай transition для тіні карток
